src/Umbraco.Web.UI.Client/src/packages/content/content/workspace/content-detail-workspace-base.ts

```diff
@@ -392,18 +392,14 @@ export abstract class UmbContentDetailWorkspaceContextBase<
 		this.#segments.setValue(data?.items ?? []);
 	}
 
-	protected override async _scaffoldProcessData(data: DetailModelType): Promise<DetailModelType> {
+	protected override async _processIncomingData(data: DetailModelType): Promise<DetailModelType> {
```

Comment:
This is a breaking change, I believe you can recreate the old method as an alias of the new method to preserve backwards compatibility in case someone calls it from the outside.


```suggestion
    /**
     * @deprecated Call `_processIncomingData` instead. `_scaffoldProcessData` will be removed in v.18.
     */
	protected override _scaffoldProcessData(data: DetailModelType): Promise<DetailModelType> {
	    return this._processIncomingData(data);
	}

	protected override async _processIncomingData(data: DetailModelType): Promise<DetailModelType> {
```